### PR TITLE
cartopy cleanup

### PR DESCRIPTION
**Pull request by darkblue-b**
remove 6MB, add 3 MB with cartopy 0.14 install

### CODE REVIEW

File: bin/base_python.sh [hunk 1]
```diff
@@ -42,7 +42,20 @@ apt-get install --yes python-gdal python-shapely python-rasterio \
...
+tar xf cartopy_0143_data.tgz
+mkdir /usr/lib/python2.7/dist-packages/cartopy/data
+chmod 755 /usr/lib/python2.7/dist-packages/cartopy/data
+cp -R cartopy_0143_data/*  /usr/lib/python2.7/dist-packages/cartopy/data/
+## TODO link to ~/data
+##=========================
+
 # Install Geospatial Python3 libraries
-apt-get install --yes python3-gdal fiona rasterio
+apt-get install --yes python3-gdal fiona rasterio python3-matplotlib
```
Comment (kalxas) [inline bin/base_python.sh]: This addition adds 12.6MB to the disk.
I think we should only keep the Python2 version of this library for 12.0

File: bin/base_python.sh [hunk 2]
```diff
@@ -42,7 +42,20 @@ apt-get install --yes python-gdal python-shapely python-rasterio \
 	python-pyshp python-descartes python-geographiclib python-kml \
 	python-cartopy python-seaborn python-networkx
 
+##======================================================
+# clean cartopy  12beta  rm 6MB, add 3MB
+rm -rf /usr/lib/python2.7/dist-packages/cartopy/tests/*
+cd /tmp; mkdir tmp_cartopy; cd tmp_cartopy
+wget -c http://download.osgeo.org/livedvd/12/cartopy/cartopy_0143_data.tgz
```
Comment (kalxas) [inline bin/base_python.sh]: How about moving this part to the Jupyter installer?

Comment (kalxas) [reply bin/base_python.sh]: actually, just leave where python-cartopy is installed so we can track

File: bin/base_python.sh (hunk 1 above)
Comment (Bakaniko) [reply bin/base_python.sh]: I think with the cleanup in PR #236 made by @darkblue-b on python (98 MB), we can allow to use some of the free space for python3 packages. Expecially if you can make spatial operation but can't render them.

**Pull request merged**